client/src/components/ShopCardModal/ShopCardItem.js

```diff
@@ -12,43 +12,44 @@ import PlusOneSharpIcon from '@material-ui/icons/PlusOneSharp';
 import ExposureNeg1SharpIcon from '@material-ui/icons/ExposureNeg1Sharp';
```

Comment:
to={`/product/${product.url}`} it's better to create some routedBuilder and not create links directly in the components